data/authorities.json

```diff
@@ -183,6 +183,27 @@
       }
     }
   },
+  "IL": {
+    "utility": {
```

Comment:
I just landed #286, which means this will need some updates: the utility IDs need to be part of that Energy Star dataset. You can see the available IDs by doing this:

- Run `yarn build`
- `sqlite3 incentives-api.db`
- `select utility_id from utilities where utility_id like 'il-%';`

In particular, you'll have to change:

- `il-corn-belt-energy-corporation` -> `il-corn-belt-energy`
- `il-mid-american-energy-company` -> `il-mid-american-energy`

Reply:
The spreadsheet-to-json script can eventually handle this but for now, you may want to amend the above to:
`select * from utilities where utility_id like 'il-%';` so it shows both the IDs and the display name.

Then change the spreadsheet so the Authority name is the same as the correct display name. spreadsheet-to-json then *should* come up with the correct ID on its own.

In the long-term we'll have validation on the spreadsheet so that you can only choose valid Authority names (or IDs) in the first place, but this change was landed a few hours ago so it'll be a bit awkward temporarily.

Reply:
How do we rectify the state level weatherization incentive IL-35, which the script is currently identifying as `il-state-of-illinois` I didn't see an equivalent "authority" in the sqllite query `select * from utilities where utility_id like 'il-%';` ?

Reply:
done